src/jwt/base64.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Borrowed from IDToken-verifier package
+ * https://github.com/auth0/idtoken-verifier/blob/master/src/helpers/base64.js
+ */
+import base64 from 'base64-js';
+
+export function padding(str) {
+  var mod = str.length % 4;
```

Comment:
Shouldn't that `4` be in a constant with a descriptive name?

Reply:
I've taken these files (and their tests) from `auth0.js` as-is, as the header on this file points out. I don't think I should be moddifying them.